### ISSUE TITLE
[24.10.0-rc2] mediatek: bpi-r4: 50% performance degradation on 10G SFP+ port

### ISSUE THREAD

**Issue by MichelHanzen** [openwrt/openwrt#17207]
### Describe the bug

Tests performed on fiber 10gb connection:
2 x10Gtek for Ubiquiti UniFi SFP+ Multimode Transceiver UF-MM-10G, 10G SFP+ SR Fibre Module, Dual LC Connector, 850nm
Fiber Patch Cable - LC to LC OM3 10Gb/Gigabit]
With openwrt 24.10-RC2
![image](https://github.com/user-attachments/assets/04a8d151-bfd2-4860-a0e8-f9a568c8141e)

With the openwrt image available on the bpi-r4 site (BPI-R4-BE1350-WIFI_MP4_0-SDK-20240620)
![image](https://github.com/user-attachments/assets/fcbb15d7-ab63-4cdf-a282-e41a08bf5af3)

### OpenWrt version

r28161-ea17e958b9

### OpenWrt release

24.10.0-rc2

### OpenWrt target/subtarget

mediatek/filogic

### Device

Bananapi BPI-R4

### Image kind

Official downloaded image

### Steps to reproduce

Run iperf3 on the two images

### Actual behaviour

speed below 5 Gbits/sec

### Expected behaviour

speed should be near 10 Gbits/sec

### Additional info

Link to the bpi-r4 image:
https://drive.google.com/file/d/1U0tnvoaxDujO1q5QNxnw0i2d93X2zi9r/view?usp=sharing

### Terms

- [X] I am reporting an issue for OpenWrt, not an unsupported fork.

**Comment by MichelHanzen** [comment]
Report updated. Sorry.

**Comment by MichelHanzen** [comment]
Thé tests I performed were made between my PC and the router on the LAN. Is bridger helpful in this case? If yes, how do I have to implement it? I guess "opkg install bridger" is not enough.

**Comment by RomanovDanila** [comment]
RSS and LRO aren't implemented in the mainline linux. Please, re-read the message above.

**Comment by MichelHanzen** [comment]
You don't answer may question and I don't understand your answer. I will close the issue.
